### PR TITLE
Initial work on a table of our courses, with grade on horizontal axis.

### PR DESCRIPTION
**Pull request by breville**
Initial work on a partial that shows our courses in a table.  For review by ed team at private URL of /educate/courses.

Entries are clickable to expand one at a time with more information.

screenshot 2016-01-25 12 03 01

screenshot 2016-01-25 12 03 11

### CODE REVIEW

**Comment by ashercodeorg** [comment]
Since the age <==> grade map is country dependent, would it make sense to specify ages rather than grades?

File: pegasus/sites.v3/code.org/public/courses.haml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    - courses.each do |course|
+      .course.break
+        - unless course[:starts] == 0
+          %div{class: "bar-#{course[:starts]}"}
+            &nbsp;
+        %div{class: "courserow bar-#{course[:ends] - course[:starts] + 1}"}
+          .title
+            =course[:name]
+            %br/
+          -# if course[:subtitle]
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/public/courses.haml]: Can we remove if not needed?

File: pegasus/sites.v3/code.org/public/courses.haml [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+---
+
+- (1..13).each do |n|
+  :css
+    .courses .bar-#{n} {
+      float: left;
+      width: #{n * 7}%;
+    }
+
+:css
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/public/courses.haml]: You know how I feel about css being in a separate file. ^_^ I'd be ok with it if it was in a course_table.haml view and that's used in several educate pages.

**Comment by tanyaparker** [comment]
In general I think this is awesome. But we should chat with the education team to see if it's correct/useful/where we'd want to use it.

**Comment by tanyaparker** [comment]
@ashercodeorg you bring up a good point. Aside from code.org and code.org/learn, pretty much all the other pegasus pages are not expected to be localized. We should create these pages in alignment with our own (U.S.) goals, and then if other countries want the same resources they would localize on their partner page i.e. uk.code.org.

File: pegasus/sites.v3/code.org/public/courses.haml (hunk 2 above)
Comment (breville) [inline pegasus/sites.v3/code.org/public/courses.haml]: That was always the plan if this emerged from prototype.  Now done. :)

**Pull request merged**